src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -233,8 +236,16 @@ public StateFlow build(FlowState.Main initialState) {
             // 3. Def1 registered, Def 2 unrepresented
             .transitionTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT)
                 .onlyIf((respondent1NotRepresented.and(respondent2NotRepresented))
+                    .or(respondent1NotRepresented.and(respondent2OrgNotRegistered.negate()))
```

Comment:
Has this been tested with 1 v 2 case where respondent 2 is not represented or not registered as well?

Reply:
I've just added more tests for this scenario, ran them and they pass